api/api/src/main/scala/uk/ac/wellcome/platform/api/models/Aggregation.scala

```diff
@@ -69,6 +69,31 @@ object Aggregations extends Logging {
       Try(License.createLicense(str)).toEither.left
         .map(err => err.getMessage)
     }
+
+  implicit class EnhancedTransformable(transformable: Transformable) {
+    def toAgg[T: Decoder]: Option[Aggregation[T]] = {
+      transformable
+        .safeTo[EsAggregation[T]]
+        .toOption
```

Comment:
I believe this will swallow errors which we would probably like to be alerted to? Maybe you are aware of this as still a draft, but just noticed it.

Reply:
Hmmm, kinda agree about the errors but we've been swallowing them [all along](https://github.com/wellcometrust/catalogue/blob/master/api/api/src/main/scala/uk/ac/wellcome/platform/api/models/Aggregation.scala#L24) and an error in parsing the JSON here would indicate that ES has broken its API response contract, which seems like something we can assume won't happen...

Reply:
Interesting that already has this behaviour. I would at the very least log this as an error (even if we still decide to return the response with the field as `null`), as probably something we would be keen to be alerted to